stories/dialogModal/index.stories.tsx

```diff
@@ -128,6 +138,9 @@ export const WithColor = () => {
         <DialogModal
           title="Heads up!"
           backgroundColor={theme.COLORS.background}
+          onClose={() => {
+            setWithColor(false);
```

Comment:
[nit] Not a big deal and I know you didn't add this, but I feel like `setWithColor` is a slightly confusing name. Maybe something like `setOpenModalWithColor`?

Reply:
<img src="https://media0.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif"/>

Reply:
Good callout! I'll make the change